### ISSUE TITLE
Augmentation dots on rests

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#673]
Augmentation dots for rests have gotten messed up recently:

![screen shot 2017-08-21 at 5 46 09 am](https://user-images.githubusercontent.com/3487289/29519933-125d3da0-8634-11e7-912e-8f3688d5d7be.png)

Also, long and breve rests do not allow augmentation dots (would be nice to have for completeness).

Augmentation dots for notes are not messed up, but long and breve augmentation dots seem a bit far from the notes compared to the other rhythms.

Test data:

```
**kern
00r.
0r.
1r.
2r.
4r.
8r.
16r.
32r.
64r.
128r.
256r.
=
00.b
0.b
1.b
2.b
4.b
8.b
16.b
32.b
64.b
128.b
256.b
=
*-
```
Transcoding to MEI:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-08-21T05:44:14" version="2.0.0-dev-369222b-dirty">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000001314656080" midi.bpm="400">
                        <staffGrp xml:id="staffgrp-0000000745583173">
                            <staffDef xml:id="staffdef-0000000396328684" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000001244635831">
                        <measure xml:id="measure-L1" n="0">
                            <staff xml:id="staff-0000000483571865" n="1">
                                <layer xml:id="layer-L1F1N1" n="1">
                                    <rest xml:id="rest-L2F1" dots="1" dur="long" />
                                    <rest xml:id="rest-L3F1" dots="1" dur="breve" />
                                    <rest xml:id="rest-L4F1" dots="1" dur="1" />
                                    <rest xml:id="rest-L5F1" dots="1" dur="2" />
                                    <rest xml:id="rest-L6F1" dots="1" dur="4" />
                                    <rest xml:id="rest-L7F1" dots="1" dur="8" />
                                    <rest xml:id="rest-L8F1" dots="1" dur="16" />
                                    <rest xml:id="rest-L9F1" dots="1" dur="32" />
                                    <rest xml:id="rest-L10F1" dots="1" dur="64" />
                                    <rest xml:id="rest-L11F1" dots="1" dur="128" />
                                    <rest xml:id="rest-L12F1" dots="1" dur="256" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L13">
                            <staff xml:id="staff-L13F1N1" n="1">
                                <layer xml:id="layer-L13F1N1" n="1">
                                    <note xml:id="note-L14F1" dots="1" dur="long" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L15F1" dots="1" dur="breve" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L16F1" dots="1" dur="1" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L17F1" dots="1" dur="2" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L18F1" dots="1" dur="4" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L19F1" dots="1" dur="8" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L20F1" dots="1" dur="16" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L21F1" dots="1" dur="32" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L22F1" dots="1" dur="64" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L23F1" dots="1" dur="128" oct="4" pname="b" accid.ges="n" />
                                    <note xml:id="note-L24F1" dots="1" dur="256" oct="4" pname="b" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed with 0053a318856d8bff7c70505e54d9b30a946336f9

**Comment by craigsapp** [comment]
Looks great:

<img width="694" alt="screen shot 2017-08-28 at 7 09 31 am" src="https://user-images.githubusercontent.com/3487289/29777177-e9c9b044-8bbf-11e7-8f71-52f578a830fe.png">

Although no dots on the breve and long rests, which might be useful for modern-notation renderings of 14th century music...

For the dots on the breve and long, they are 20% closer to the noteheads than for the smaller notes.  Is there a reason for that?  Here is a graphic analysis where there are 13 pixels between the breve note and dot, and 16 pixels between the whole note and dot:

<img width="1511" alt="screen shot 2017-08-28 at 7 17 28 am" src="https://user-images.githubusercontent.com/3487289/29777572-5fc88314-8bc1-11e7-8724-4e8352c90626.png">
